browser/src/docdispatcher.ts

```diff
@@ -303,13 +324,15 @@ class Dispatcher {
 			app.map.onFormulaBarBlur();
 			app.map.formulabarBlur();
 			app.map.formulabarSetDirty();
+			app.allowExit();
```

Comment:
`allowExit()` is already called by fieldBlur() which is invoked by onFormulaBarBlur()
anyway,  I added comments on the fieldBlur() definition, too